### PR TITLE
Updated accessiblity for Inputs

### PR DESCRIPTION
**Pull request by jwoo-msft**
## Related Issue
reference #4898 

## Description
Fixed the VoiceOver issue for iOS. Input's label and input field were treated as disparate units during the VoiceOver, and the input field did not provide the information contained in the input label. 

## How Verified
How you verified the fix, including one or all of the following: 
1. All Input cards in 1.3 were tested and verified.

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/microsoft/AdaptiveCards/pull/4951)

### CODE REVIEW

File: source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRInputLabelView.mm [hunk 1]
```diff
@@ -87,6 +87,7 @@ - (instancetype)initInputLabelView:(ACRView *)rootView acoConfig:(ACOHostConfig
         }
 
         self.label.attributedText = attributedLabel;
+        self.labelText = attributedLabel;
 
         std::string errorMessage = inputBlck->GetErrorMessage();
         if (!errorMessage.empty()) {
```
Comment (shalinijoshi19) [inline source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRInputLabelView.mm:93]: @jwoo-msft is this the case where we need to append the error message to the label as well? Are we making hte change here to tackle #4912 also?

File: source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRInputTimeRenderer.mm [hunk 2]
```diff
@@ -37,6 +37,10 @@ - (UIView *)render:(UIView<ACRIContentHoldingView> *)viewGroup
     ACRDateTextField *field = [[ACRDateTextField alloc] initWithTimeDateInput:timeInput dateStyle:NSDateFormatterNoStyle];
 
     ACRInputLabelView *inputLabelView = [[ACRInputLabelView alloc] initInputLabelView:rootView acoConfig:acoConfig adptiveInputElement:timeInput inputView:field viewGroup:viewGroup dataSource:nil];
+
+    field.isAccessibilityElement = YES;
```
Comment (shalinijoshi19) [inline source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRInputTimeRenderer.mm:41]: Question: We seem to be repeating this across teh various input elements - isnt it possible to update the base input class with these changes so all derived elements as well as custom elements potentially, are able to get these changes applied automatically?

File: source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRInputLabelView.mm (hunk 1 above)
Comment (jwoo-msft) [reply source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRInputLabelView.mm]: @shalinijoshi19, I had created a separate branch off this branch for #4912. I just merged it into this branch for CR purposes.

File: source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRInputTimeRenderer.mm (hunk 2 above)
Comment (jwoo-msft) [reply source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRInputTimeRenderer.mm]: With the merge with the branch for #4912, this is resolved.

File: source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRChoiceSetViewDataSourceCompactStyle.mm [hunk 3]
```diff
@@ -218,11 +245,12 @@ - (void)tableView:(UITableView *)tableView didSelectRowAtIndexPath:(NSIndexPath
 - (BOOL)validate:(NSError **)error
 {
     if (self.isRequired) {
-        BOOL result = !(!_userSelectedTitle || [_userSelectedTitle isEqualToString:_defaultString]);
-        return result;
+        _isValid = !(!_userSelectedTitle || [_userSelectedTitle isEqualToString:_defaultString]);
+        return _isValid;
     }
+    _isValid = YES;
     // no need to validate
```
Comment (RebeccaAnne) [inline source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRChoiceSetViewDataSourceCompactStyle.mm:252]: >// no need to validate [](start = 4, length = 22)

[nit] I'd move this comment above the _isValid = YES; line

**Review by RebeccaAnne — APPROVED** [review]
:shipit:

**Pull request merged**